autocomplete_light/fixtures/autocomplete_model_test_case.json

```diff
@@ -67,7 +71,8 @@
     "fields": {
         "username": "Elton",
         "groups": [],
-        "email": "elton@example.com"
+        "email": "elton@example.com",
+	"ordering": 10
```

Comment:
I don't want to be "that guy" and complain about indentation here, it's, after all, JSON and it doesn't matter ^^